_includes/markdown/JavaScript.md

```diff
@@ -155,6 +155,74 @@ On all new projects you should be using up to date JavaScript methodologies comb
 
 Some older projects that have not yet been upgraded may not have the capability to use the most modern techniques, but it is still important to have processes in place that allow us to grow the technology stack as a project matures. In these cases, you should still follow best practice recommendations even if the newest patterns are not yet available to you.
 
+### Secure Your Code
+
+In JavaScript, we often have to insert new elements with dynamic attributes and content into the DOM. A common way to do this is to use the ```innerHTML``` method like so:
+
+```javascript
+const myElement = document.getElementById( 'myElement' );
```

Comment:
Maybe change `myElement` to `element` to differentiate it from the elements that you're creating. It's has a pretty mild effect here, but in the more verbose `textContent` example it might add a little bit of clarity.